backend/alembic_revision.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python3
+
+# SPDX-FileCopyrightText: Copyright DB Netz AG and the capella-collab-manager contributors
+# SPDX-License-Identifier: Apache-2.0
+
+"""
+Create a new revision file for Alembic.
+
+Usage:
+
+   ./alembic_revision.py "short summary"
+
+"""
+import logging
+import os
+import sys
+
+from alembic import command
+from alembic.config import Config
+from sqlalchemy import create_engine
+from testcontainers.postgres import PostgresContainer
+
+import capellacollab.config.models
+import capellacollab.extensions.backups.ease.models
+import capellacollab.extensions.modelsources.git.models
+import capellacollab.extensions.modelsources.t4c.models
+import capellacollab.notices.models
+import capellacollab.projects.capellamodels.models
+import capellacollab.projects.models
+import capellacollab.projects.users.models
+import capellacollab.sessions.models
+import capellacollab.settings.modelsources.git.models
+import capellacollab.tools.models
+import capellacollab.users.models
+from capellacollab.core.database import migration
+
+log = logging.getLogger(__name__)
+
+
+def main(summary: str):
+
+    with PostgresContainer("postgres:14.1") as postgres:
+        database_url = postgres.get_connection_url()
+        log.info("Setting up database connection")
+        engine = create_engine(database_url)
+
+        log.info("Initializing database")
+        migration.migrate_db(engine)
+
+        os.environ["ALEMBIC_SQLALCHEMY_URL"] = database_url
+
+        alembic_cfg = Config("./alembic.ini")
+        command.revision(alembic_cfg, message=summary, autogenerate=True)
+
+
+if __name__ == "__main__":
+    logging.basicConfig(level=logging.DEBUG)
+
+    main(summary=sys.argv[1])
```

Comment:
I don't get the point of this script. Alembic compares the current database state with how it should look like (by looking to the code) when autogenerating the revision. When you spawn a database before and initialize it with the state from the code, then there is exactly no difference between the current database structure and the code - and alembic doesn't detect the changes.

My workflow to run the migrations is to setup a database, initialize it with the current state and then make the changes to the code. When I'm done with it, I run `alembic revision --autogenerate`. Because my database structure has the old version, alembic picks up the changes.